### ISSUE TITLE
New arcana voicelines tied to wrong items in Cosmetic section

### ISSUE THREAD

**Issue by hunbogi** [jeffhill/Dota2#1547]
### Description

With the introduction with new voicelines for the heroes with Arcanas some voicelines are tied to the Arcana itself instead of the Voiceslin comsetic that accompanied the Arcana. Ogre Magi for example. Taking the voicepackage off or putting it back on has no result on what voicelines are used with the dota+ voicelines in-game. However if you take the Arcana off the voicelines revert to the old usual self. I assume this is a bug but not intentional, because if so it would make the Voicelines cosmetic slot largely redundant and take away players choice and making the arcana pay to lose. 
I have not checked other arcanas.
You can replicate this in demo for Ogre Magi.

**Comment by gudni-maker** [comment]
Yes I would also like that particular change! Give us choice which voice lines we want to use! There is already a simple way to implement this with some arcanas having voice slots. Why not make the voice lines depend on which you have equipped? The voice lines in the chat wheel of course!
